index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you use a class selector to style your heading.

## Example
`<h1 class="header">Hello Microverse!</h1>`